SpatialGDK/Source/SpatialGDKFunctionalTests/Public/SpatialFunctionalTest.h

```diff
@@ -154,6 +154,13 @@ class SPATIALGDKFUNCTIONALTESTS_API ASpatialFunctionalTest : public AFunctionalT
 	UFUNCTION(BlueprintCallable, Category = "Spatial Functional Test", meta = (ToolTip = "Check is the Actor has it's authority delegated to a specific Server Worker. \n\nKeep in mind that currently this functionality only works in single layer Load Balancing Strategies, and your Default Load Balancing Strategy needs to implement ISpatialFunctionalTestLBDelegationInterface."))
 	bool HasActorDelegation(AActor* Actor, int& WorkerId, bool& bIsPersistent);
 
+	// # Actor Interest APIs
+	UFUNCTION(BlueprintCallable, Category = "Spatial Functional Test", meta = (ServerWorkerId = "1", ToolTip = "Allow Server Worker to always have interest in an Actor, even it if moves outside it's interest area.\n\nNote that this only works if you are using USpatialFunctionalTestGridLBStrategy."))
+	void AddActorInterest(int32 ServerWorkerId, AActor* Actor);
+
+	UFUNCTION(BlueprintCallable, Category = "Spatial Functional Test", meta = (ServerWorkerId = "1", ToolTip = "Counterpart to AddInterestOverActor().\n\nNote that this only works if you are using USpatialFunctionalTestGridLBStrategy."))
```

Comment:
```suggestion
	UFUNCTION(BlueprintCallable, Category = "Spatial Functional Test", meta = (ServerWorkerId = "1", ToolTip = "Counterpart to AddActorInterest().\n\nNote that this only works if you are using USpatialFunctionalTestGridLBStrategy."))
```
? But I also like AddInterestOverActor.

Reply:
Initially I had called the function that, but then because the other ones are AddActorDelegation I thought to "normalize them". If you think it works better, I can rename them. What would you suggest?

Reply:
I don't have a strong preference, so keeping consistency will be easiest.